### PR TITLE
fixed:quickstart page responsiveness

### PR DESCRIPTION
**Pull request by theajmalrazaq**
just fixed the quickstart page responsivess it is subpart of issue #25:mobile view is broken
Screenshot_select-area_20241019124711
Screenshot_select-area_20241019124722

### CODE REVIEW

**Review by seveibar — CHANGES_REQUESTED** [review]
Change the number of grid cols not the display style (this is a good use case for grid, switching to flex isnt as good)

**Comment by theajmalrazaq** [comment]
@seveibar  Done!!

**Comment by seveibar** [comment]
So it looks like youve changed the design on lg screens.

im mobile today so its harder to check work. Can you introduce a playwright snapshot test (see playwright-tests directory) for xs, md and lg sizes. This will also benefit the project because others will need to show their snapshot for future modifications

**Comment by theajmalrazaq** [comment]
@seveibar, this is how it looks on both devices. On large devices, there will be 3 columns, and on mobile, there will be 1 column. Furthermore, I am working on Playwright tests for 3 screen sizes: large, medium, and extra small, and I’m fixing the AI page UI. I will create their PR tomorrow. Also, I will work on setting the screen to ensure it is fixed across mobile. With all these updates, issue #25 will be closed. For now, please merge this.

Screenshot_select-area_20241020002600
Screenshot_select-area_20241020002546

File: src/pages/quickstart.tsx [hunk 1]
```diff
@@ -159,11 +161,13 @@ export const QuickstartPage = () => {
           <h2 className="text-xl font-semibold mb-4 mt-12">
             Start from a Template
           </h2>
-          <div className="grid grid-cols-3 gap-4">
+          <div className="grid grid-cols-1 md:grid-cols-3 gap-4">
```
Comment (seveibar) [inline src/pages/quickstart.tsx:164]:
```suggestion
          <div className="grid grid-cols-1 md:grid-cols-3 lg:grid-cols-4 gap-4">
```

File: src/pages/quickstart.tsx [hunk 2]
```diff
@@ -73,14 +73,16 @@ export const QuickstartPage = () => {
 
         <div className="mb-8">
           <h2 className="text-xl font-semibold mb-4">Start Blank Snippet</h2>
-          <div className="grid grid-cols-4 gap-4">
+          <div className="grid grid-cols-1 md:grid-cols-3 gap-4">
```
Comment (seveibar) [inline src/pages/quickstart.tsx:76]:
```suggestion
          <div className="grid grid-cols-1 md:grid-cols-3 lg:grid-cols-4 gap-4">
```

File: src/pages/quickstart.tsx [hunk 3]
```diff
@@ -46,7 +46,7 @@ export const QuickstartPage = () => {
           {isLoading ? (
             <div>Loading...</div>
           ) : (
-            <div className="grid grid-cols-4 gap-4">
+            <div className="grid grid-cols-1 md:grid-cols-3 gap-4">
```
Comment (seveibar) [inline src/pages/quickstart.tsx:49]:
```suggestion
            <div className="grid grid-cols-1 md:grid-cols-3 lg:grid-cols-4 gap-4">
```

**Review by seveibar — COMMENTED** [review]
See comments, then good to go

**Comment by theajmalrazaq** [comment]
@seveibar doing it right away.

**Comment by theajmalrazaq** [comment]
@seveibar Done with this!

**Review by seveibar — APPROVED** [review]

**Pull request merged**